src/store/currentTodo.ts

```diff
@@ -0,0 +1,40 @@
+import { Todo } from '../types/Todo';
+
+type SetSelectedTodo = {
+  type: 'SELECT_TODO',
+  payload: Todo | null,
+};
+
+type UnsetSelectedTodo = {
+  type: 'UNSELECT_TODO',
+  payload: null,
+};
```

Comment:
```suggestion
type SetSelectedTodo = {
  type: 'todo/SELECT_TODO',
  payload: Todo | null,
};

type UnsetSelectedTodo = {
  type: 'todo/UNSELECT_TODO',
  payload: null,
};
```
suggestion: add prefixes